t/01_run.t

```diff
@@ -164,7 +165,7 @@ subtest 'process execute()' => sub {
   is $p->exit_status, 100,         'able to retrieve function return';
 
   $p = Mojo::IOLoop::ReadWriteProcess->new(
-    sleeptime_during_kill => 0.1,
+    sleeptime_during_kill => $interval,
```

Comment:
Not sure if this is intentional but the $interval and the previous value differ. here and in L136

Reply:
Good catch but it is intentional, see the commit message.